test/syscalls/linux/fork.cc

```diff
@@ -215,6 +215,7 @@ TEST_F(ForkTest, PrivateMapping) {
   EXPECT_THAT(Wait(child), SyscallSucceedsWithValue(0));
 }
 
+#ifdef __x86_64__
```

Comment:
What is the problem here? Is fork() not working for ARM?

Reply:
No, it's because the arm64 doesn't support the CPUID instruction which is used in the GetCPUVendor() call.

Reply:
I think you need to add a comment that explains this.

Reply:
@avagin Done, thanks!